api/src/main/java/io/minio/ServerSideEncryption.java

```diff
@@ -248,8 +243,16 @@ public final void destroy() throws DestroyFailedException {
     }
   }
 
-  static final class ServerSideEncryptionKms extends ServerSideEncryption {
+  /**
+   * Create a new server-side-encryption object for encryption at rest (a.k.a. SSE-S3).
+   *
+   * @return an instance of ServerSideEncryption implementing SSE-S3
+   */
+  public static ServerSideEncryption atRest() {
```

Comment:
We should keep AWS S3 naming conventions.

Reply:
I haven't changed any name here. Whatever renames I made have `deprecated` annotation.

Reply:
Oh sry, my bad.